frost-core/src/frost/keys.rs

```diff
@@ -29,6 +29,35 @@ use super::compute_lagrange_coefficient;
 pub mod dkg;
 pub mod repairable;
 
+/// Sum the commitments from all participants in a distributed key generation
+/// run into a single group commitment.
+#[cfg_attr(feature = "internals", visibility::make(pub))]
+pub(crate) fn sum_commitments<C: Ciphersuite>(
+    commitments: &[&VerifiableSecretSharingCommitment<C>],
+) -> Result<VerifiableSecretSharingCommitment<C>, Error<C>> {
+    let mut group_commitment = vec![
+        CoefficientCommitment(<C::Group>::identity());
+        commitments
+            .get(0)
+            .ok_or(Error::IncorrectCommitment)?
```

Comment:
I feel like a more appropriate error here would be an InvalidCommitment as IncorrectCommitment is more about a commitment as it relates to a participant. Just a thought but not crucial.

Reply:
I think having both Invalid and Incorrect would be confusing, it doesn't seem clear what would be their difference. I went for reusing IncorrectNumberOfCommitments in 7965e24 which seems to be the actual issue here (either `commitments` is empty, or the one commitment vector has a different number of commitments). What do you think?